contracts/src/StakeTable.sol

```diff
@@ -515,6 +524,82 @@ contract StakeTable is AbstractStakeTable, Ownable, InitializedAt {
         emit UpdatedConsensusKeys(msg.sender, node.blsVK, node.schnorrVK);
     }
 
+    // Delegation functions
+
+    // @notice creates a pool and configure the initial fee
+    // TODO: docs
+    function createPool(
+        BN254.G2Point memory blsVK,
```

Comment:
curious if hotshot is aware of the ethereum address of a validator and if so if it should be added here (since the address is meant to be the unique identifier and the bls key can change)

Reply:
It's currently assumed that it's `msg.sender`. I expect the confirmation layer to get it from the emitted event. Do you think we should do something different?